packages/front-generator/src/common/studio/studio-integration.ts

```diff
@@ -1,6 +1,9 @@
-import * as http from "http";
+import fetch from 'node-fetch'
 
 const DEFAULT_INTEGRATION_PORT = 48561;
+export const ERR_STUDIO_NOT_CONNECTED = '' +
+  'Can\'t establish connection to CUBA Studio. Please open CUBA Studio Intellij and enable Old Studio integration. ' +
```

Comment:
In Studio 13+ the checkbox is called `Enable integration`, lets call it correspondingly here (old studio integration sounds weird)